src/JavascriptVM/vm.ts

```diff
@@ -23,8 +25,28 @@ export class BlocklyInterpreter {
         console.log("VM > " + text);
       });
 
+      const setDcMotorPower = interpreter.createNativeFunction(
+        (port: number, power: number) => {
+          this.pause = true;
+          callbacks.onSetDcMotorPower(port, power);
+        }
+      );
+
+      const isSensorTouchPushed = interpreter.createNativeFunction(
+        (port: number, power: number) => {
+          this.pause = true;
```

Comment:
Would be helpful to have some comments on the purpose of pause flag
The power parameter is not used. Is it needed?

Reply:
i think its not required. can likely be removed